playground/App.vue

```diff
@@ -0,0 +1,57 @@
+<template>
+  <div>
+    <h1 class="row">
+      <PhPhosphorLogo /> Phosphor Icons Vue 3
+    </h1>
+    <div class="row">Total icon count: {{ total }}</div>
+    <div v-for="icon in allIcons" class="row">
+      <span class="name">{{ icon.name }}</span>
+      <component :is="icon" size="32" />
+      <component :is="icon" size="24" weight="thin" :mirrored="true" />
+      <component :is="icon" size="32" weight="light" />
+      <component :is="icon" size="48" />
+      <component :is="icon" size="32" weight="bold" :color="c1" />
+      <component :is="icon" size="32" weight="fill" :color="c2" />
+      <component :is="icon" size="32" weight="duotone" :color="c3" />
+    </div>
+  </div>
+</template>
+
+<script setup>
+import { PhPhosphorLogo } from "../src";
+import * as icons from "../src";
+
+const ALIASES = {
+  "file-dashed": "file-dotted",
+  "file-magnifying-glass": "file-search",
+  "folder-dashed": "folder-dotted",
+  "folder-simple-dashed": "folder-simple-dotted",
+  pulse: "activity",
+  seal: "circle-wavy",
+  "seal-check": "circle-wavy-check",
+  "seal-question": "circle-wavy-question",
+  "seal-warning": "circle-wavy-warning",
+  "text-b": "text-bolder",
+};
```

Comment:
I would love to import this directly from `core/bin` but I couldn't figure out how to get CJS interop working.